lib/features/login/data/network/interceptors/authorization_interceptors.dart

```diff
@@ -49,11 +49,11 @@ class AuthorizationInterceptors extends QueuedInterceptorsWrapper {
     _token = newToken;
     _configOIDC = newConfig;
     _authenticationType = AuthenticationType.oidc;
-    log('AuthorizationInterceptors::setTokenAndAuthorityOidc: TOKEN_INITIAL = $newToken');
+    log('AuthorizationInterceptors::setTokenAndAuthorityOidc: INITIAL_TOKEN = ${newToken?.token} | EXPIRED_TIME = ${newToken?.expiredTime}');
   }
 
   void _updateNewToken(TokenOIDC newToken) {
-    log('AuthorizationInterceptors::_updateNewToken: NEW_TOKEN = $newToken');
+    log('AuthorizationInterceptors::_updateNewToken: NEW_TOKEN = ${newToken.token} | EXPIRED_TIME = ${newToken.expiredTime}');
```

Comment:
Why line 52 contains `${newToken?.expiredTime}` _(with `?`)_ and this line contains `${newToken.expiredTime}`. Error risk ?

Reply:
`newToken` is non-null parameter. We pre-check it in line 101-105, moreover, we will throw exception in case we can not execute `refreshToken` success with idP (line 97-99)